starwars/sets/CAD.txt

```diff
@@ -123,4 +123,4 @@ Elite Magnaguard	CAD	CADsub_Elite_Magnaguard	D	Subordinate	Character - Separatis
 Gunship Crew Soldier	CAD	CADsub_Gunship_Crew_Soldier	L	Subordinate	Character - Republic Clone Soldier		50	2	2	S			Pay 0 Force -> Intercept	
 Infantry B1 Droid	CAD	CADsub_Infantry_B1_Droid	D	Subordinate	Ground/Character - Trade Federation Droid		40	2	2	S			Critical Hit 1	
 Prowler Patrol Droid	CAD	CADsub_Prowler_Patrol_Droid	N	Subordinate	Character - Republic Droid		60	1	2	S			Your opponent's Characters lose Stealth and can't use Intercept.	
-Separatist Escape Pod	CAD	CADsub_Separatist_Escape_Pod	D	Subordinate	Space - Separatist Device		10	0	2	S				
\ No newline at end of file
+Separatist Escape Pod	CAD	CADsub_Separatist_Escape_Pod	D	Subordinate	Space - Separatist Device		10	0	2	S				
```

Comment:
I don't see a difference in these two lines. I wonder why it was flagged as a change.

Reply:
Haha... I wondered if anyone else would notice that. I'm not sure why if flags it as changed. Probably just an oddity with comparing text files.

If Lackey would have used the .tsv file extension for these files it would have been a lot better.